### PR TITLE
Add render template service

### PR DESCRIPTION
**Pull request by oleh-momot**

### CODE REVIEW

**Comment by StyleT** [comment]
`registry/tests/common.ts → registry/tests/common.spec.ts`

revert this change. Helpers shouldn't have `.spec.` suffix

File: registry/server/templates/services/renderTemplate.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+                ...includes,
+                [include]: include,
+            };
+        }
+    }, Promise.resolve({}));
+
+    return Object.keys(includesData).reduce((
+        template: any,
+        include: any,
+    ) => template.split(include).join(includesData[include]), template);
```
[the remark this answers is not in the document]
Comment (oleh-momot) [reply registry/server/templates/services/renderTemplate.ts]: Added

File: registry/server/templates/services/renderTemplate.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+            const [key, value] = attribute.split('=');
+
+            return {
+                ...attributes,
+                [key]: value.split('\"').join(''),
+            };
+        }, {}),
+    }), {});
+
+    const includesData = await Object.keys(includesAttributes).reduce(async (
```
[the remark this answers is not in the document]
Comment (oleh-momot) [reply registry/server/templates/services/renderTemplate.ts]: Rewrote

File: registry/server/templates/services/renderTemplate.ts [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+// TODO Need to add types
+async function renderTemplate(template: any) {
+    const includesRegExp = /(?:\<include)(?:\s*(?:(?:\w*)\=\"(?:[^\"]*)\")\s*)*(?:\s*(?:\/\>|\>\s*(?:.*)\s*\<\/include\>))/gmi;
+    const includesAttributesRegExp = /((?:\w*)\=\"(?:[^\"]*)\")/gmi;
+
+    const includesAttributes: any = [...template.matchAll(includesRegExp)].reduce((
+        includes: any,
+        [include]: any,
+    ) => ({
```
[the remark this answers is not in the document]
Comment (oleh-momot) [reply registry/server/templates/services/renderTemplate.ts]: Moved

File: registry/server/templates/services/renderTemplate.ts [hunk 4]
```diff
@@ -0,0 +1,67 @@
+import axios from 'axios';
+
+// TODO Need to add types
+async function renderTemplate(template: any) {
+    const includesRegExp = /(?:\<include)(?:\s*(?:(?:\w*)\=\"(?:[^\"]*)\")\s*)*(?:\s*(?:\/\>|\>\s*(?:.*)\s*\<\/include\>))/gmi;
+    const includesAttributesRegExp = /((?:\w*)\=\"(?:[^\"]*)\")/gmi;
+
+    const includesAttributes: any = [...template.matchAll(includesRegExp)].reduce((
```
[the remark this answers is not in the document]
Comment (oleh-momot) [reply registry/server/templates/services/renderTemplate.ts]: Replaced

File: registry/server/templates/services/renderTemplate.ts [hunk 5]
```diff
@@ -0,0 +1,67 @@
+import axios from 'axios';
+
+// TODO Need to add types
+async function renderTemplate(template: any) {
+    const includesRegExp = /(?:\<include)(?:\s*(?:(?:\w*)\=\"(?:[^\"]*)\")\s*)*(?:\s*(?:\/\>|\>\s*(?:.*)\s*\<\/include\>))/gmi;
```
[the remark this answers is not in the document]
Comment (oleh-momot) [reply registry/server/templates/services/renderTemplate.ts]: Removed

File: registry/.mocharc.json [hunk 6]
```diff
@@ -0,0 +1,7 @@
+{
+    "spec": "**/*.spec.ts",
+    "require": "ts-node/register",
+    "timeout": 5000,
+    "exclude": "node_modules/**/*",
+    "exit": true
```
Comment (StyleT) [inline registry/.mocharc.json]: rm `"exit": true`

**Pull request merged**